ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+This is a template helping you to create an issue which can be processes as quickly as possible. Feel free to add additional information or remove not relevant points if you do not need them.
+### Please state the information for your system
+
+OpenCV | Your operation system | (if needed) Compiler / Cmake
+------------ | ------------- | ------------- | -------------
+3.x or 2.4 | Windows 10 64 Bit / Ubuntu 15.04 32 bit / Mac OS X El Capitan 10.11.3 | Visual Studio 2015 / GCC 5.3, Cmake 3.5
```

Comment:
I fiddled a bit around with lists and tables but yeah you are probably right :+1: